### PR TITLE
Dependabot Fixes - static html sample

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What
Update azure-communicaiton-common and nanoid dependencies.

# Why
Fixes dependabot issues for static html composites

# How Tested

Built all flavors to see if anything failed

### CODE REVIEW

File: samples/StaticHtmlComposites/package.json [hunk 1]
```diff
@@ -20,7 +20,9 @@
   "license": "MIT",
   "dependencies": {
     "@azure/communication-react": "1.0.1-beta.2",
-    "@azure/communication-common": "1.0.0",
+    "@azure/communication-common": "1.1.0",
+    "@azure/communication-calling": "1.3.2-beta.1 || 1.3.2",
```
Comment (anjulgarg) [inline samples/StaticHtmlComposites/package.json:24]: Why do we need to add these deps explicitly?

File: packages/chat-stateful-client/package.json [hunk 2]
```diff
@@ -33,8 +33,8 @@
     "@internal/acs-ui-common": "1.0.1-beta.2",
     "events": "~3.3.0",
     "immer": "9.0.6",
-    "nanoid": "~3.1.20",
-    "@azure/communication-common": "1.0.0"
+    "nanoid": "~3.1.31",
+    "@azure/communication-common": "1.1.0"
```
Comment (JamesBurnside) [inline packages/chat-stateful-client/package.json:37]: nit: keep these deps alphabetized

File: samples/tests/package.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
   "license": "MIT",
   "dependencies": {
     "@azure/communication-react": "1.0.1-beta.2",
```
Comment (JamesBurnside) [inline samples/tests/package.json:22]: if this uses `@azure/communication-react` this should likely also specify the `@azure/communication-calling` and `chat` deps like the static html sample

File: packages/react-composites/package.json [hunk 4]
```diff
@@ -62,7 +62,7 @@
     "events": "~3.3.0",
     "immer": "9.0.6",
     "memoize-one": "~5.2.1",
-    "nanoid": "~3.1.20",
+    "nanoid": "~3.1.31",
```
Comment (JamesBurnside) [inline packages/react-composites/package.json:65]: be sure to mention changes like this in the PR description if they are intended

Comment (JamesBurnside) [reply packages/react-composites/package.json:65]: PR specifies just `Update azure-communicaiton-common package`

**Review by JamesBurnside — APPROVED** [review]

File: packages/chat-stateful-client/package.json (hunk 2 above)
Comment (dmceachernmsft) [reply packages/chat-stateful-client/package.json:37]: kk will go fix that!

File: samples/StaticHtmlComposites/package.json (hunk 1 above)
Comment (dmceachernmsft) [reply samples/StaticHtmlComposites/package.json:24]: common needed a update for some of its child dependencies and calling and chat were moved out of being peer dependencies so they needed to be declared here too.

Comment (prprabhu-ms) [reply samples/StaticHtmlComposites/package.json:24]: This is correct. An oversight, all our samples need to declare chat/calling as deps.

**Review by prprabhu-ms — APPROVED** [review]

**Pull request merged**